packages/babel/src/generation/generators/flow.js

```diff
@@ -178,6 +178,14 @@ export function NullableTypeAnnotation(node, print) {
 }
 
 /**
+ * Prints NumberLiteralTypeAnnotation, prints value.
+ */
+
+export function NumberLiteralTypeAnnotation(node, print, parent) {
+  this.Literal(node, print, parent);
```

Comment:
Actually just realised that this should really be just exporting the `Literal` method verbatim instead of calling it. So something like:

```javascript
import { Literal } from "./types";
export { Literal as NumberLiteralTypeAnnotation };
```